### PR TITLE
Add vmvx test configuration in tflitehub

### PR DESCRIPTION
**Pull request by hcindyl**
* Enable --config=vmvx to test vmvx HAL backend.
* Add the `%config_flag` in lit.cfg.py to enable the flag in lit test.
  * Tag the failing test
* Add vmvx in GH action regression test.

### CODE REVIEW

File: tflitehub/lightning_fp16_test.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# RUN: %PYTHON %s
+# RUN: %PYTHON %s %config_flag
+# XFAIL: vmvx
```
Comment (hcindyl) [inline tflitehub/lightning_fp16_test.py:2]: @jpienaar This test fails only with vmvx backend. Should I file a GH issue to track it?

Comment (jpienaar) [reply tflitehub/lightning_fp16_test.py:2]: Yes - I'm pretty sure I know what this is from (sub-32 bit has some gaps VMVX side - its mostly in the conversions to the underlying storage and loading back, these aren't a priority for us at the moment but please flag if blockers).

Comment (hcindyl) [reply tflitehub/lightning_fp16_test.py:2]: It is not a blocker for our work. I will just tag it as XFAIL for now. :)

**Comment by hcindyl** [comment]
> Can you link to CI runs before/after this change? How much time does this add to the workflow run?

https://github.com/hcindyl/iree-samples/actions/runs/4623709067/jobs/8177804640

It doubles the regression test run time (I turn off the hugetest in b1c9b16).

**Review by ScottTodd — APPROVED** [review]

**Pull request merged**